### DISCUSSION TITLE
CHow to continue using old OsmAND+?

### DISCUSSION THREAD

**Discussion by josefmoellers** [osmandapp/OsmAnd#15945] [General]
Hi,
I'm using Osmand ever since I have a smartphone, which is ever since there was a Fairphone on the market.
I immediately also purchased the "+" plugin to d/l as many maps as I want.
This still works on my Phone and also on my (old) tablet.
However, the other day I dropped my tablet and had to purchase a new one which I'm in the process of installing.
Probably due to the change in licensing, I am unable to get my "osmand plus" which I hat purchased before. 
Do I have to shell out the new license fee or is there a way to continue using my old "osmand plus"?
Thanks,
Josef

**Comment by stephan75** [comment]
Can you find some help on this page?

https://osmand.net/docs/user/purchases

**Reply by jmoellers** [reply]
Unfortunately not. It says that I should either re-install OsmAnd+ using the same account as previously, but this asks for 24,99€ or to do it from inside OsmAnd (Menu → Settings → Purchases → Restore purchases) but that says that it restored the purchase but nothing changes.

**Comment by dmpr0** [comment]
Hi! 
All purchases are linked to the account and App Store you are using for making this purchase. If you purchase OsmAnd+ in Google Play, make sure you use the same Google account.
You can contact us at support@osmand.net with your Google Account email address or transaction number (you can try to find it by searching "GPA" in mailbox)

**Comment by josefmoellers** [comment]
Hello Dmytro,

thanks for replying.

I checked if I could find "GPA" somewhere in my mailbox but the only hit
was your mail ;-)

My email address is "josef.moellers(at)gmx.de" and this address has been
used from the beginning in the Goggle Play Store, aamof I never had any
other email address (I did have work-related addresses but they were
never used for my mobile devices). Strangely enough: on my FairPhone in
the AppStore's "Budget & Verlauf", I also see only my purchases since
May 2021, nothing before that.

Thanks in advance,

Josef

**Comment by jmoellers** [comment]
I forgot to finish this: I had been wrong in assuming that the GMX email address was the only one used. Rather, I had registered my mobile devices using the GMAIL address automagically generated for me.
Logging into the Play Store using those credentials allowed me to install OsmAnd+ for free!
I apologize for the late reply,
Josef
